dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/CompanyApi.kt

```diff
@@ -367,4 +367,27 @@ interface CompanyApi {
         produces = ["application/json"],
     )
     fun getCompanyInfo(@PathVariable("companyId") companyId: String): ResponseEntity<CompanyInformation>
+
+    /**
+     * A method to check if a companyId is valid
+     * @param companyId the identifier
+     */
+    @Operation(
+        summary = "Checks that a companyId exists.",
+        description = "Checks that a companyId exists.",
+    )
+    @ApiResponses(
+        value = [
+            ApiResponse(responseCode = "200", description = "Successfully checked that companyId exists."),
+            ApiResponse(responseCode = "404", description = "Successfully checked that companyIdr does not exist."),
+        ],
```

Comment:
companyId statt companyIdr. Ich find die description auch wiedersprüchlich. Vllt.: Check was successful, but companyId does not exist

Reply:
Typo changed. 
Bei dem Check ist es so, der ist halt auch successfull wenn er herausfindet, dass es die Firma nicht gibt. Ein nicht successfull request wäre, wenn der mit einem 500 Fehler fehlschlägt.

Reply:
Das versteh ich. Mein Vorschlag hatte sich auf den zweiten Fall bezogen. Wir können es aber auch so lassen.

Reply:
Dann ändere ich das zu: Check was successfull. Company is not known by dataland.
Passt das für dich?

Reply:
Ja